### PR TITLE
Fix/ repeat duration

### PR DESCRIPTION
**Pull request by eheinrich**
Use the repeat_duration to figure out percent completed for repeat type configurations

### CODE REVIEW

File: observation_portal/requestgroups/request_utils.py [hunk 1]
```diff
@@ -122,20 +122,32 @@ def get_airmasses_for_request_at_sites(request_dict, is_staff=False):
...
+                completed_time += (
+                        configuration_status.summary.end - configuration_status.summary.start
+                ).total_seconds()
+            configuration_total_time = configuration_status.configuration.repeat_duration
+        else:
+            if has_summary:
+                completed_time += configuration_status.summary.time_completed
+            for instrument_config in configuration_status.configuration.instrument_configs.all():
+                configuration_total_time += instrument_config.exposure_count * instrument_config.exposure_time
+        total_time += configuration_total_time
```
Comment (jnation3406) [inline observation_portal/requestgroups/request_utils.py]: I might be misreading this, but it seems like you could just have one variable here that is zero in the beginning and adds onto the total time the whole way through? I probably wrote it the other way for no good reason before

Comment (eheinrich) [reply observation_portal/requestgroups/request_utils.py]: Do you mean doing something like adding on the `total_time` variable directly, and getting rid of the intermediate `configuration_total_time` variable?

Comment (jnation3406) [reply observation_portal/requestgroups/request_utils.py]: yeah that's what I meant. It seemed like an unnecessary extra variable

Comment (eheinrich) [reply observation_portal/requestgroups/request_utils.py]: :+1: Agreed, done!

**Pull request merged**